fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/PartialDataV2RequestHandler.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.handlers;
+
+import com.yahoo.bard.webservice.config.SystemConfig;
+import com.yahoo.bard.webservice.config.SystemConfigProvider;
+import com.yahoo.bard.webservice.druid.model.query.DruidAggregationQuery;
+import com.yahoo.bard.webservice.web.DataApiRequest;
+import com.yahoo.bard.webservice.web.responseprocessors.ResponseProcessor;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * A request handler that builds responses which filter partial data V2.
+ */
+public class PartialDataV2RequestHandler implements DataRequestHandler {
+
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+
+    private final DataRequestHandler next;
+
+    /**
+     * Constructor.
+     *
+     * @param next  Next Handler in the chain
+     */
+    public PartialDataV2RequestHandler(@NotNull DataRequestHandler next) {
```

Comment:
While we're working on the v2 extension to the Partial Data capability, the "v2" part of this is not very informative as to what this class does or why it's interesting. So, instead of continuing to talk about v2, can we instead use names and descriptions for things that indicate what they are doing? (ie. what would we have called this if it was implemented right away, instead of now)